pkg/health/http_test.go

```diff
@@ -134,4 +132,14 @@ func TestHTTPHealthChecker(t *testing.T) {
 			t.Errorf("Expected %v, got %v", test.health, health)
 		}
 	}
+	container := api.Container{
+		LivenessProbe: &api.LivenessProbe{
+			HTTPGet:        &api.HTTPGetAction{},
+			TimeoutSeconds: 1,
+		},
+	}
+	health, _ := hc.HealthCheck("test", "", api.PodStatus{PodIP: "0.0.0.0"}, container)
```

Comment:
How do we know this exercises the timeout path?  Should you instead have a second test case that has a HandlerFunc() that just sleeps for 5 seconds?